src/components/NcRichContenteditable/NcRichContenteditable.vue

```diff
@@ -32,7 +32,9 @@ Try mentioning user @Test01 or inserting emoji :smile
 ```vue
 <template>
 	<div>
+		<NcButton class="show-slash-button" @click.stop="showSlashCommands('/')">Slash commands</NcButton>
```

Comment:
Can we move it to a new example to not start component docs with it?

For example, to have a separate example with all the public methods.